src/Control/Map_Edit.cpp

```diff
@@ -77,22 +77,28 @@ bool CMapView::SnappingIsOn(bool shift)
 // ************************************************************
 bool CMapView::HandleOnMouseMoveShapeEditor(int x, int y, long nFlags)
 {
+
+	tkLayerSelection behavior;
+	_shapeEditor->get_SnapBehavior(&behavior);
+
+	double projX, projY;
+	bool snapped = FindSnapPointCore(x, y, &projX, &projY);
+	if (!snapped) {

```

Comment:
Similar to earlier, FindSnapPointCore returns a VARIANT_BOOL, and you're setting it into a bool variable, and then 'NOT'ing it.  'snapped' should be changed to VARIANT_BOOL, OR perhaps instead, we should change FindSnapPointCore to return a bool, since it is an internal function and is more easily handled with the standard bool data type.